person.rb

```diff
@@ -20,6 +21,10 @@ def correct_name
     @name
   end
 
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] This method is for creating a rental given a book and date. Since the rental is added upon creation, this method needs to take that into account. Kindly refactor the method to create a rental that associates this person with the book through rental.

  ```
  def add_rental(date, book)
     Rental.new(date, book, self)
  end
  ```